### DISCUSSION TITLE
How to globally configure TinyMCE (WYSIWYG) for data object? (Version 11.1.4)

### DISCUSSION THREAD

**Discussion by dariusmann** [pimcore/pimcore#16799] [Q&A / Support]
The only thing which is documented is how to configure [EDITABLE WYSIWYG](https://pimcore.com/docs/platform/Pimcore/Documents/Editables/WYSIWYG/). But not how to configure DATA OBJECT WYSIWYG.

**Comment by skoch98** [comment]
Hi, take a look at my answer in a newer post. I guess this will help you: https://github.com/pimcore/pimcore/discussions/16841#discussioncomment-9039211

^Sebastian
